crates/loro-internal/src/encoding/value.rs

```diff
@@ -170,6 +173,7 @@ pub enum Value<'a> {
 #[derive(Debug)]
 pub enum FutureValue<'a> {
     // The future value cannot depend on the arena for encoding.
+    EmptyTreeTrash(Arc<Vec<TreeID>>),
```

Comment:
Currently users must upgrade to be able to parse `EmptyTrash` Op, so can we put this Op in `Value` to reduce the encoding size?

Reply:
Do we have other options?

Reply:
What are the possible designs and their trade-offs?

Reply:
I moved it to `Value`